README.md

```diff
@@ -10,24 +10,58 @@ Python Battery Mathematical Modelling solves continuum models for batteries, usi
 
 ## How do I use PyBaMM?
 
-PyBaMM comes with a number of [detailed examples](examples/README.md), hosted here on github. In addition, there is a [full API documentation](http://pybamm.readthedocs.io/), hosted on [Read The Docs](readthedocs.io).
+PyBaMM comes with a number of [detailed examples](examples/README.md), hosted here on
+github. In addition, there is a [full API documentation](http://pybamm.readthedocs.io/),
+hosted on [Read The Docs](readthedocs.io).
 
-## How can I install PyBaMM?
-
-The safest way to install PyBaMM is to do so within a virtual environment ([introduction to virtual environments](https://realpython.com/python-virtual-environments-a-primer/))
+## How can I obtain & install PyBaMM?
 
 You'll need the following requirements:
 
 - Python 3.5+
-- Python libraries: `numpy` `scipy` `pandas` `matplotlib`
+- Git (`git` package on Ubuntu distributions)
+- Python libraries: `venv` (`python3-dev` package on Ubuntu distributions)
+
+The first step is to get the code by cloning this repository
+
+```bash 
+git clone https://github.com/pybamm-team/PyBaMM.git
+cd PyBaMM
+```
+
+The safest way to install PyBaMM is to do so within a virtual environment ([introduction
+to virtual environments](https://realpython.com/python-virtual-environments-a-primer/)).
+To create a virtual environment `env` within your current directory type:
+
+```bash
+python3 -m venv env
+```
 
-These can easily be installed using `pip`. To do this, first make sure you have the latest version of pip installed:
+You can then "activate" the environment using:
+
+```bash 
+source env/bin/activate 
+```
+
+Now all the calls to pip described below will install PyBaMM and its dependencies into
+the environment `env`. When you are ready to exit the environment and go back to your
+original system, just type:
+
+```bash 
+deactivate 
+```
+
+PyBaMM has the following python libraries as dependencies: `numpy`, `scipy`, `pandas`,
+`matplotlib`. These, along with PyBaMM, can easily be installed using `pip`. To do this,
+first make sure you have the latest version of pip installed:
 
 ```bash
 pip install --upgrade pip
 ```
 
-Then navigate to the path where you downloaded PyBaMM to, and install both PyBaMM and its dependencies by typing:
+Then navigate to the path where you downloaded PyBaMM to (you will already be in the
+correct location if you followed the instructions above), and install both PyBaMM and
+its dependencies by typing:
 
 ```bash
 pip install .
```

Comment:
I know this is from previous commits but will `pip install .` install pybamm so that it in the system path and therefore visible from the notebooks?

Reply:
yea, they should. The `-e` just allows them to edit the source, and have the package update to reflect this. If you don't put it in, then any changes you make don't update the package you have installed